src/Tools/CompilerBenchmarks/CompilerBenchmarks.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeIdentifiers>$(RoslynPortableRuntimeIdentifiers)</RuntimeIdentifiers>
+    <IsShipping>false</IsShipping>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\Compilers\CSharp\Test\Emit\Microsoft.CodeAnalysis.CSharp.Emit.UnitTests.csproj" />
```

Comment:
BuildBoss going to be unhappy about this. It's going to say it's illegal to reference unit test projects. We will need to add an exception for this case.

Reply:
@jaredpar I could either add IVT to another assembly, or fix up BuildBoss. Do you have a preference on which one? I stuck them together because I've been increasingly thinking how weird it is that we subdivide benchmarks and unit tests.

Reply:
Imagine that we hadn't artificially divided our unit test assemblies and there was just a single assembly for C# and VB respectively. How would you want to define benchmarks in that world? I'm a bit curious why we'd want to merge them with our normal tests given they take two different harnesses to run. 

---
In reply to: [210435918](https://github.com/dotnet/roslyn/pull/29323#discussion_r210435918) [](ancestors = 210435918)

Reply:
I'll separate them for now. To answer your question, I wonder whether or not they should be different test harnesses and this should actually be a part of XUnit.